wrangler-api/src/main/java/io/cdap/wrangler/api/Row.java

```diff
@@ -141,6 +156,9 @@ public Row setValue(int idx, Object value) {
   public Row add(String name, Object value) {
     columns.add(name);
     values.add(value);
+    if (nameIndex != null && name != null) {
+      nameIndex.put(name.toLowerCase(Locale.ENGLISH), columns.size());
```

Comment:
`columns.size()-1` ?

Reply:
good catch, is this a 0-based or 1-based index?

Reply:
Done